packages/block-editor/src/components/alignment-toolbar/index.js

```diff
@@ -30,6 +30,7 @@ const DEFAULT_ALIGNMENT_CONTROLS = [
 
 const POPOVER_PROPS = {
 	position: 'bottom right',
+	isPrimary: true,
```

Comment:
Wonder if we should use a more descriptive name here because I don't feel primary / secondary is what should determine the weight. Maybe something like "isFloating"? Or "isBlockUi" since these are used for blocks only? Just a thought.

Reply:
I'm super duper open to changing this, and now is the best time to do so. Which is your preference? We could also go in a different direction, like `isEmphasized`?

Reply:
Maybe `isAlternate` for now?